common.mk

```diff
@@ -50,6 +50,11 @@ LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.Date=$(
 LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.SHA=$(BUILD_SHA)'
 LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.Version=$(BUILD_VERSION)'
```

Comment:
Do we need a TODO comment to remove these once got obsolete?

Reply:
Good question.  I'm not sure if these will be removed or changed.  The new `buildinfo` are just for plugins, so I expect the existing `buildinfo` will be moved to some other location.

I'm expecting/hoping that the refactoring of the `pkg/v1/buildinfo` will know to correct the lines above as they should still be needing them.

Does that makes sense, or did I misunderstand the next steps?

Reply:
Yes, sounds good:)